lib/lotus/action/cookie_jar.rb

```diff
@@ -131,6 +137,7 @@ def extract(env)
         hash   = env[COOKIE_HASH_KEY] ||= {}
         string = env[HTTP_HEADER]
 
+        return if string && string.include?(RACK_SESSION_KEY)
```

Comment:
I'm not entirely sure:-(

Perhaps we can skip setting a cookie with key  ```rack.session``` in ```CookieJar``` altogether? Or only when ```sessions``` is enabled with ```:cookie```?